server/storage/rethinkdb_models.go

```diff
@@ -13,7 +13,7 @@ const (
 
 var (
 	// TufFilesRethinkTable is the table definition of notary server's TUF metadata files
```

Comment:
No issues, I'll send in a fresh PR with this edited. My bad, I completely missed it as well